src/test/java/teammates/ui/webapi/UpdateFeedbackSessionActionTest.java

```diff
@@ -94,6 +99,240 @@ protected void testExecute() {
 
         assertNotNull(response.getCreatedAtTimestamp());
         assertNull(response.getDeletedAtTimestamp());
+
+        Map<String, Long> expectedStudentDeadlines = new HashMap<>();
+        expectedStudentDeadlines.put("student3InCourse1@gmail.tmt", 1809126000000L);
+        expectedStudentDeadlines.put("student4InCourse1@gmail.tmt", 1809126000000L);
+        expectedStudentDeadlines.put("student5InCourse1@gmail.tmt", 1809126000000L);
```

Comment:
Do the timings mean anything? Would be cleaner if we gave the numbers a name that represents its value. 

e.g.
```
Long timeRepresentsNow = TimeHelper.formatInstant("date");
```

Then we can use it as follows
```
..., timeRepresentsNow
..., TimeHelperExtension.plus(...) // can assign to variable if used in multiple places
```